docs/Thatonecloud.md

```diff
@@ -0,0 +1,3 @@
+# demo
+
+- Not typing a lot on a phone...
```

Comment:
You forgot to add your Name, Position and Location. But I guess that is hard working from a phone. :smile: